app/gateway/resource-sizing-guidelines.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "{{site.base_gateway}} resource sizing guidelines"
+content_type: reference
+layout: reference
+
+products:
+    - gateway
+
+works_on:
+    - on-prem
+
+min_version:
+    gateway: 3.4.x
+
+tags:
+    - performance
+
+breadcrumbs:
+    - /gateway
+
+description: "See Kong's recommended resource allocation sizing guidelines for {{site.base_gateway}} based on configuration and traffic patterns."
+
+related_resources:
+  - text: Performance benchmark
+    url: /
+  - text: Performance benchmark how to
+    url: /
+  - text: Cluster reference
+    url: /
+---
+
+{{site.base_gateway}} is designed to handle large volumes of request
+traffic and proxying requests with minimal latency. This reference offers recommendations on sizing for
+resource allocation based on expected {{site.base_gateway}} configuration and
+traffic patterns.
+
+{{site.base_gateway}} measures performance in the following dimensions:
+
+| Performance dimension | Measured in | Performance limited by... | Description |
+|-----------------------|-------------|-------------|
+| Latency | Microseconds or milliseconds | *Memory-bound*<br>Add more database caching memory to increase | The delay between the downstream client sending a request and receiving a response. Increasing the number of Routes and Plugins in a {{site.base_gateway}} cluster increases the amount of latency that's added to each request. |
+| Throughput | Seconds or minutes |*CPU-bound*<br>Scale {{site.base_gateway}} vertically or horizontally to increase | The number of requests that {{site.base_gateway}} can process in a given time span |
```

Comment:
```suggestion
| Throughput | Seconds or minutes |*CPU-bound*<br>Scale {{site.base_gateway}} vertically or horizontally to increase throughput | The number of requests that {{site.base_gateway}} can process in a given time span |
```
This might sound like common sense (what else are you increasing?), but this reads more clear to me.